packages/patternfly-4/react-core/src/components/ClipboardCopy/ClipboardCopy.d.ts

```diff
@@ -1,16 +1,18 @@
-import { FunctionComponent, HTMLProps, MouseEvent, ReactNode, HTMLDivElement } from 'react';
-import { OneOf } from '../../typeUtils';
+import { FunctionComponent, HTMLProps, MouseEvent, ReactNode } from 'react';
+import { OneOf } from '../../helpers/typeUtils';
 import { PopoverPosition } from '../Popover';
 
-export const ClipboardCopyVariant = {
-  inline: 'inline',
-  expansion: 'expansion'
-};
+export enum ClipboardCopyVariant {
+  inline = 'inline',
+  expansion = 'expansion'
+}
 
 export interface ClipboardCopyProps extends HTMLProps<HTMLDivElement> {
   className?: string;
   hoverTip?: string;
   clickTip?: string;
+  'aria-label'?: string;
+  'toggle-aria-label'?: string;
```

Comment:
Since we are still prerelease, do you mind changing this prop here and in the JS file to `toggleAriaLabel`.  This is the naming convention we have been using for non HTML aria attributes.